gr-blocks/grc/blocks_null_sink.block.yml

```diff
@@ -21,7 +21,7 @@ parameters:
     dtype: int
     default: '1'
     hide: part
--   id: bus_conns
+-   id: bus_structure_sink
```

Comment:
explicitly calling out as bus_structure_sink, as that is what the underlying property is.  Also a good pattern for blocks that have both sink and source